components/javascript/src/main/java/org/datacleaner/beans/script/JavaScriptTransformer.java

```diff
@@ -52,9 +52,7 @@
 	private static final Logger logger = LoggerFactory
 			.getLogger(JavaScriptTransformer.class);
 
-	public static enum ReturnType {
-		STRING, NUMBER, BOOLEAN;
-	}
+	public enum ReturnType {STRING, NUMBER, BOOLEAN}
```

Comment:
Not sure we have a convention on this type of thing - but if we have, is this it? I would normally use the old format of defining the inner enum. In general I would always have newlines after/before curly braces.